### PR TITLE
[3.7] bpo-11233: Create availability directive for documentation (GH-…

### PR DESCRIPTION
**Pull request by csabella**
…9692)

 Replace "Availability: xxx" with ".. availability:: xxx" in the doc.
 Original patch by Georg Brandl.

 Co-Authored-By: Georg Brandl <georg@python.org>
 (cherry picked from commit 2d6097d027e0dd3debbabc702aa9c98d94ba32a3)

 Co-authored-by: Cheryl Sabella <cheryl.sabella@gmail.com>

https://bugs.python.org/issue11233

### CODE REVIEW

File: Doc/using/cmdline.rst [hunk 1]
```diff
@@ -834,7 +834,7 @@ conflict.
    order to force the interpreter to use ``ASCII`` instead of ``UTF-8`` for
    system interfaces.
 
-   Availability: \*nix
+   .. availability:: \*nix.
```
Comment (1st1) [inline Doc/using/cmdline.rst:837]: Can we use "Unix" everywhere and not "*nix"?

Comment (csabella) [reply Doc/using/cmdline.rst:837]: I didn't change any of the original text attached to Availability in order to just make this ticket about the new directive.  If you want, I can open another ticket to clean up things like this.

Comment (vstinner) [reply Doc/using/cmdline.rst:837]: > If you want, I can open another ticket to clean up things like this.

Yes, please don't do two things in the same PR. master also has this syntax.

**Review by vstinner — APPROVED** [review]
LGTM.

**Pull request merged**